api/base/filters.py

```diff
@@ -24,20 +24,29 @@
 def sort_multiple(fields):
     fields = list(fields)
     def sort_fn(a, b):
-        while fields:
-            field = fields.pop(0)
+        rev = 1
```

Comment:
Let's make rev something more readable. How about rename it into `sort_direction`, then define `FORWARD = 1` and `REVERSE = -1`. Then you can set `sort_direction = REVERSE` and `return 1 * sort_direction`, for example.